jest.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     '**/__integration__/',
     'packages/amplify-e2e-core/',
     'packages/amplify-e2e-tests/',
-    'packages/amplify-console-integration/',
+    'packages/amplify-console-integration-tests/',
```

Comment:
We dont need this here. Otherwise these tests will be run by jest-vscode plugin and would end up clogging your account

Reply:
is that exactly why we need to put it into the `testPathIgnorePatterns`?

Reply:
last I checked testIgnorePattern did not seemed to work for monrepo. Can you confirm ? If it doesnt please remove this from here.

Reply:
My Bad, I did not realize this is in test-ignore pattern